### DISCUSSION TITLE
Return - Reason Why I Was Out

### DISCUSSION THREAD

**Discussion by RealDealJinx** [DEMOLITIONDON96/Demolition-Engine#105] [General (Friday Night Chattin')]
So i have return from me being grounded

The reason is a very long story so hold up (may make no sense)

Me and @Wulsy were on call on discord
My mom walked in the room and started asking questions on why I was on call with Wulsy
Since Wulsy didn't took it seriously and did what she asked, I then got grounded
She then said that Wulsy could be a pedophile when he wasn't doing nothing that leads to being a pedophile

so yeah
conclusion, WULSY NO PEDO
but I am ready to join back to the team

also I deleted my discord account

**Comment by Theoyeah** [comment]
![image](https://user-images.githubusercontent.com/97792861/170455425-db6564eb-0abd-4b50-8a97-77d3983ef83f.png)
Think that theres people that never touched discord (like me)

**Comment by ghost** [comment]
Lets raid Wulsy in his discord server.

**Reply by ghost** [reply]
-____-

**Comment by ghost** [comment]
OH fuck what have I done
